lib/scholars_archive/embargoes/embargo_releaser.rb

```diff
@@ -0,0 +1,20 @@
+module ScholarsArchive::Embargoes
+  class EmbargoReleaser
+    extend Hyrax::EmbargoHelper
+    
+    def self.expire_embargoes
+      expired_embargoes = self.find_all_expired_embargoes
+      expired_embargoes.each do |embargo|
+        work = ActiveFedora::Base.find(embargo.solr_document[:id])
+        Hyrax::Actors::EmbargoActor.new(work).destroy
```

Comment:
can you add some logging right before this line to print out details about the work that it's about to expired, for troubleshooting purposes?